.github/workflows/test.e2e.yml

```diff
@@ -4,11 +4,14 @@ on:
   schedule:
     - cron: '0 0 * * *'
   push:
+    branches:
```

Comment:
because before this I found that in an open PR, when I push a new commit to a feature/fix branch, `e2e_tests` job will be triggered by both `push` and `pull_request`, so it will run 2 actions that are the same, we only need to run this job once in this case

Reply:
Got it